src/Components/BrainstormCard/BrainstormCard.test.tsx

```diff
@@ -8,10 +8,16 @@ describe('BrainstormCard component', () => {
   const mockProps = {
     id: 1,
```

Comment:
Is this Prettier making these changes? I don't mind reviewing files, but this seems a little superfluous